### ISSUE TITLE
Newest update multiplayer stuck at 99% syncing!

### ISSUE THREAD

**Issue by Tubezki** [s4t4n/FS22_CVT_Addon#15]
Hi again,

I wrote you about the rpm issue but also found out with my friend that we are not able to play multiplayer with latest update, he gets stuck at 99% loading screen and syncing game with the players is the text in the screen.

**Comment by DrewCZ** [comment]
Hi @Tubezki, something bad in the log file? Could you try to push both logs here?

**Comment by Tubezki** [comment]
Log was clean and we got it working after trying later which is really weird but there is still problems with the rpm and stuff with latest update: https://www.youtube.com/watch?v=n4-07GyeC2E

**Comment by DrewCZ** [comment]
Hi, the sync issue in MP is caused by GS servers, which have a problem - same for me, and thousands of other people. If you wish to have more info, watch their discord. So, issue #15 is not related to the CVT addon. 
Your posted YT is related to other issue #14 I think.

**Comment by Tubezki** [comment]
Hi @DrewCZ I tested only leaving VCA and CVT addon to only mods with the new holland, and it works update before the new CVT addon with shop configuration. Its a problem with VCA and CVT addon with the Keep Speed option from VCA, and this mod has been developed VCA compatible before but now the Keep Speed bugs the RPM with both mods enabled VCA and CVT. This is how far I have digged the code and been testing many hours now.
